src/main/scala/com/fulcrumgenomics/alignment/Alignment.scala

```diff
@@ -82,4 +120,55 @@ case class Alignment(query: Array[Byte],
 
   /** One based closed coordinate of the end of the alignment on the query sequence. */
   def targetEnd: Int = targetStart + cigar.lengthOnTarget - 1
+
+  /**
+    * Generates a padded text representation of the alignment for visualization. The returned
+    * sequence will consist of three lines as follows (minus the labels on the left):
+    *
+    * query : ACGTGAACTGACT-ACTGTATGCG
+    * align : |||||  |||||| ||||||||.|
+    * target: ACGTG--CTGACTGACTGTATGGG
+    *
+    * @param matchChar the character to use in the alignment line when the bases match
+    * @param mismatchChar the character to use in the alignment line when the bases mismatch
+    * @param gapChar the character to use in the alignment line when one of the sequences is gapped
+    * @return Three lines representing the alignment
+    */
+  def paddedString(matchChar: Char = '|', mismatchChar: Char = '.', gapChar: Char = ' '): Seq[String] = {
```

Comment:
I did think about that.  My thinking was that a `Seq[String]` would work better if we ever wanted to do something more sensible for longer alignments where we want to wrap them, instead of creating one big long line for each.